src/debugger/Debugger.h

```diff
@@ -37,14 +37,15 @@ namespace Escargot {
 #define ESCARGOT_DEBUGGER_NO_STACK_TRACE_RESTORE (reinterpret_cast<ExecutionState*>(0x1))
 #define ESCARGOT_DEBUGGER_MAX_VARIABLE_LENGTH 128
 
+class Context;
 class Object;
 class String;
 class ExecutionState;
 class ByteCodeBlock;
 class InterpretedCodeBlock;
 
 class Debugger : public gc {
-    friend Debugger* createDebugger(const char* options, bool* debuggerEnabled);
+    friend void createDebugger(const char* options, Context* context);
```

Comment:
Just a suggestion.
How about defining `createDebugger` function as static member function of `Debugger` class instead of global-scoped function?
This approach could remove the `friend` keyword too.

Reply:
Good idea!